djitellopy/tello.py

```diff
@@ -191,7 +191,8 @@ def udp_state_receiver():
                     continue
 
                 data = data.decode('ASCII')
-                drones[address]['state'] = Tello.parse_state(data)
+                data = Tello.parse_state(data)
+                drones[address]['state'] = {'timestamp': int(time.time() * 1_000_000), **data}
```

Comment:
I am developing a package for ros based on djitellopy library. To publish the status of the tello, it is enough to compare the time of receiving the packet from the socket. Why do you think it is optimal to create an object to compare time?

Reply:
Yes. In python datetime comparasion is a breeze, since you can simply subtract two datetimes and get a timedelta which abstracts all the complexity away from you.
Normally I would also be against using more libraries, but datetime is a built-in, which does not need to be installed, so it should be ok to use

Reply:
Based on the current field name, the epoch time is saved. What do you think the name of the new field to store datetime object should be?